internal/store/engine_state.go

```diff
@@ -172,7 +172,7 @@ func (e EngineState) RelativeTiltfilePath() (string, error) {
 }
 
 func (e EngineState) IsEmpty() bool {
-	return len(e.ManifestTargets) == 0 && e.GlobalYAML.Empty()
+	return len(e.ManifestTargets) == 0 && e.GlobalYAML.Name == ""
```

Comment:
This feels like a weird isEmpty check, but I'm OK with it.

Reply:
i agree! tho i think this will eventually go away as a separate field, so hopefully it will stop feeling weird soon!